tests/scripts/run-xunit-perf.cmd

```diff
@@ -18,11 +18,14 @@ setlocal
   set USAGE_DISPLAYED=
   set SHOULD_UPLOAD_TO_BENCHVIEW=
   set BENCHVIEW_PATH=
+  set COLLECTION_FLAGS=stopwatch
```

Comment:
I mean we can't have both be the case when we call run-xunit-perf.cmd with no parameters.  Personally, I would prefer that we just run the stopwatch when you call it blank.  If people want to use the tooling to do local runs stopwatch should cover what they want 99% of the time.  When they don't it's easy to specify what counters to include.